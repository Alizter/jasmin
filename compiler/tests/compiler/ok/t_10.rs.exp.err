Processed file tests/compiler/ok/t_10.mil
