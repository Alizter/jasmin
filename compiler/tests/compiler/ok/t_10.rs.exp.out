>> renum:
#[macro_use] extern crate jasmin;
param n : b64;

pub fn test() {
    var! {
        i: inline (b64)
        x: stack! (b64[(n: b64)]);
    }
    code! {
        x: stack! (b64[(n: b64)])[jc!(0)] = jc!(0);
        for i: inline (b64) in jc!(0)..(n: b64) {
            if ((((n: b64) == jc!(10)) && ((n: b64) == jc!(10))) || true) {
                x: stack! (b64[(n: b64)])[(n: b64) - jc!(1)] += jc!(1);
            } else {
              x: stack! (b64[(n: b64)])[(n: b64) - jc!(1)] += jc!(1);
            }
        }
    }
}


total transformation time: REMOVED_TIME

total time (with save/print): REMOVED_TIME
